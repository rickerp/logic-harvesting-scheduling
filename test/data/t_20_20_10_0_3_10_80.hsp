80
3
2 9 2 8 9 10 3 5 5 5 10 4 3 5 8 6 8 8 6 1 4 3 2 7 5 4 7 7 7 6 5 4 1 10 7 5 6 2 4 10 6 7 8 9 3 3 10 10 1 2 4 10 10 7 5 8 2 2 7 9 5 10 7 8 8 4 6 9 1 3 5 6 6 3 7 2 5 8 9 1
2 2 4
7 1 3 4 5 10 15 21
4 2 4 5 6
6 1 2 3 5 9 24
5 2 3 4 6 13
10 3 5 7 13 14 18 34 36 37 45
9 6 8 13 14 18 19 34 36 37
4 7 13 14 18
7 4 10 15 21 24 30 46
8 2 9 11 15 16 21 22 30
4 10 12 15 16
5 11 13 16 18 23
7 5 6 7 8 12 14 17
5 6 7 8 13 19
6 2 9 10 11 16 22
12 10 11 12 15 17 21 22 23 26 27 32 35
5 13 16 18 23 32
9 6 7 8 12 17 19 23 28 32
7 7 14 18 20 23 34 37
4 19 23 28 34
9 2 9 10 16 22 24 25 30 46
6 10 15 16 21 23 26
9 12 16 17 18 19 20 22 27 32
6 4 9 21 25 29 46
5 21 24 26 30 39
7 16 22 25 27 31 35 47
8 16 23 26 28 31 32 33 35
6 18 20 27 34 37 45
3 24 30 39
11 9 10 21 25 29 31 35 38 46 54 60
8 26 27 30 32 35 39 40 48
9 16 17 18 23 27 31 33 35 56
6 27 32 34 35 42 43
16 6 7 19 20 28 33 35 36 41 42 43 49 52 53 56 63
11 16 26 27 30 31 32 33 34 36 40 71
11 6 7 34 35 37 42 43 49 52 53 55
6 6 7 19 28 36 44
6 30 39 46 54 60 71
7 25 29 31 38 40 47 48
6 31 35 39 41 47 48
6 34 40 42 47 48 63
11 33 34 36 41 43 47 48 49 52 56 63
9 33 34 36 42 44 47 48 49 53
8 37 43 45 47 48 49 50 51
6 6 28 44 47 50 51
12 9 21 24 30 38 47 48 54 60 61 71 72
17 26 39 40 41 42 43 44 45 46 48 49 52 54 55 56 60 61
15 31 39 40 41 42 43 44 46 47 49 52 54 55 56 67
14 34 36 42 43 44 47 48 50 52 53 55 56 57 67
6 44 45 49 51 53 57
7 44 45 50 53 57 59 68
14 34 36 42 47 48 49 53 55 56 62 63 66 67 76
11 34 36 43 49 50 51 52 56 57 67 80
8 30 38 46 47 48 55 60 62
10 36 47 48 49 52 54 56 61 62 68
15 32 34 42 47 48 49 52 53 55 57 63 66 67 70 74
6 49 50 51 53 56 58
6 57 59 63 67 76 80
4 51 58 64 68
8 30 38 46 47 54 61 69 71
8 46 47 55 60 62 69 71 72
7 52 54 55 61 63 69 72
12 34 41 42 52 56 58 62 64 65 69 72 73
4 59 63 67 68
4 63 66 69 73
8 52 56 65 67 69 73 74 78
10 48 49 52 53 56 58 64 66 68 70
7 51 55 59 64 67 70 77
9 60 61 62 63 65 66 70 71 72
5 56 67 68 69 75
7 35 38 46 60 61 69 72
8 46 61 62 63 69 71 73 79
7 63 65 66 72 74 78 79
5 56 66 73 75 78
5 70 74 76 78 80
6 52 58 75 77 78 80
4 68 76 78 80
8 66 73 74 75 76 77 79 80
4 72 73 78 80
7 53 58 75 76 77 78 79
1 7 3 10 10 1 9 2 4 2 2 1 4 5 1 4 10 2 10 7 10 4 4 9 1 6 7 7 5 1 1 5 7 3 7 8 6 7 10 9 8 3 9 3 10 10 7 1 3 8 7 2 4 3 2 6 10 10 2 5 10 2 1 8 6 9 8 1 5 9 1 5 3 10 7 2 1 5 3 3
3 1 6 6 3 10 1 3 9 4 9 1 5 1 10 2 10 7 3 6 5 5 10 10 4 7 1 6 1 3 10 5 4 6 2 8 5 4 2 5 7 10 5 3 3 7 5 2 3 9 9 10 3 9 9 9 7 9 4 9 4 4 4 9 1 5 8 7 9 10 1 7 9 8 10 1 4 4 4 8
4 3 7 6 3 7 6 9 8 10 7 1 5 2 1 5 9 8 1 9 7 3 5 8 10 4 10 3 9 4 1 2 8 9 10 2 6 5 10 3 6 8 5 10 10 5 6 10 4 6 8 1 9 2 10 10 8 9 3 6 4 5 10 1 3 1 2 10 7 3 2 3 1 8 4 2 2 10 1 6
0
