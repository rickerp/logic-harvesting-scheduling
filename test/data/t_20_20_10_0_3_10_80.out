437
22 5 10 12 17 19 24 26 28 33 36 39 41 44 54 57 61 64 65 70 74 76 79
17 4 6 8 15 21 23 31 38 42 50 55 59 67 69 73 75 77
20 1 3 9 11 13 18 20 22 25 27 29 37 45 48 53 58 60 62 68 78
0
